src/Microsoft.DotNet.Build.Tasks.TargetFramework.Sdk/src/AddRuntimeOSPropertyTask.cs

```diff
@@ -9,10 +9,10 @@ namespace Microsoft.DotNet.Build.Tasks.TargetFramework.Sdk
 {
     public class GenerateRuntimeOSPropsFile : BuildTask
     {
-        public  const string RuntimeOSProperty = "RuntimeOS";
+        public const string RuntimeOSProperty = "RuntimeOS";
 
         [Required]
-        public string RuntimePropsFilePath {get ; set; }
+        public string RuntimePropsFilePath { get ; set; }
```

Comment:
(Nit) Stray space, almost there 🙂 

```suggestion
        public string RuntimePropsFilePath { get; set; }
```